.buildkite/commands/complete-code-freeze.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+
+RELEASE_NUMBER=$1
+
+if [[ -z "${RELEASE_NUMBER}" ]]; then
+    echo "Usage $0 <release number>"
+    exit 1
+fi
+
+echo '--- :git: Configure Git for release management'
+.buildkite/commands/configure-git-for-release-management.sh
+
+echo '--- :git: Checkout release branch'
+.buildkite/commands/checkout-release-branch.sh "$RELEASE_NUMBER"
+
+echo '--- :ruby: Setup Ruby tools'
+install_gems
+
+echo "--- :swift: Set up Swift Packages"
+install_swiftpm_dependencies
```

Comment:
Is SPM needed for this pipeline? I didn't think `generate_strings_file_for_glotpress` or `update_appstore_strings` would need the packages to be present or else I would think Cocoapods would be required as well.

Reply:
Ah, you are right. Must have come across from a repo that either needed it or erroneously called it.

I'll need to chase that, too...

Reply:
I see that WordPress iOS has it in the `complete-code-freeze.sh`, but it looks like it is needed there for the library localizations?